### DISCUSSION TITLE
"Checking for quick fixes..." - autocompletion is randomly broken

### DISCUSSION THREAD

**Discussion by sylvainmouquet** [microsoft/pylance-release#6157] [Q&A]
Hello,

When i am starting VSCode, the autocompletion works but after a random period, the autocompletion is broken and i would like to fix it.
Do you know if it's possible to debug the autocompletion or to find logs about this plugin ?
Thanks

**Comment by Daraan** [comment]
In VS Code Terminal view go to Output select Python Language Server. This will show you the logs.

You can modify the logging level by setting / modifying the following setting:

> Enable trace logging by adding "python.analysis.logLevel": "Trace" to your settings.json configuration file.
